dandiapi/analytics/models.py

```diff
@@ -4,7 +4,12 @@
 
 class ProcessedS3Log(models.Model):
     class Meta:
-        unique_together = ['name', 'embargoed']
```

Comment:
I didn't know `models.UniqueConstraint` is advised to be used over `unique_together`. https://docs.djangoproject.com/en/4.1/ref/models/options/#unique-together. We should remove this from the other places it's used.